portfolio/src/main/webapp/index.html

```diff
@@ -5,16 +5,37 @@
     <title>My Portfolio</title>
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
+    <style>
+
```

Comment:
extra line

Reply:
My bad

Reply:
no problem! that's what reviews are for :D